src/__tests__/userChangemakerPermissions.int.test.ts

```diff
@@ -0,0 +1,292 @@
+import request from 'supertest';
+import { app } from '../app';
+import {
+	createChangemaker,
+	createOrUpdateUserChangemakerPermission,
+	loadUserChangemakerPermission,
+} from '../database';
+import { expectTimestamp, loadTestUser } from '../test/utils';
+import {
+	mockJwt as authHeader,
+	mockJwtWithAdminRole as authHeaderWithAdminRole,
+} from '../test/mockJwt';
+import { keycloakUserIdToString, Permission } from '../types';
+import { NotFoundError } from '../errors';
+
+describe('/users/changemakers/:changemakerId/permissions/:permission', () => {
+	describe('PUT /', () => {
```

Comment:
Can we have a test for adding an already existing / duplicate permission?

Reply:
Great catch, and for the record adding this test exposed a bug 🎉